aioresponses/core.py

```diff
@@ -55,6 +62,12 @@ def __init__(self, url: Union[str, Pattern],
         self.content_type = content_type
         self.response_class = response_class or ClientResponse
         self.repeat = repeat
+        self.reason = reason
+        if self.reason is None:
+            try:
+                self.reason = http.RESPONSES[self.status][0]
+            except Exception:
```

Comment:
I think this is to broad `try ... except`
Can we change that into `except (KeyError, IndexError):`
or
`http.RESPONSES.get(self.status, [''])[0]`
?